flixpy/flix/extension/client.py

```diff
@@ -239,6 +239,17 @@ def status(self) -> types.Status:
 
         return status
 
+    @property
+    def selected_panels(self) -> list[types.PanelSelection]:
+        if not self.online:
+            return []
+        if self.project.sequence_revision is None:
+            return []
+        if len(self.panel_browser_status.revision_status.panel_selection) <= 0:
+            return []
+
+        return self.panel_browser_status.revision_status.panel_selection
```

Comment:
```suggestion
    def selected_panels(self) -> list[types.PanelSelection]:
        if self.online and self.project.sequence_revision:
            return self.panel_browser.status.revision_status.panel_selection
        return []
```
No need to check the length of `panel_selection` separately as far as I can see since if it's 0 then returning it will return an empty list anyway